charts/dapr/charts/dapr_operator/templates/dapr_operator_deployment.yaml

```diff
@@ -78,6 +78,23 @@ spec:
         - name: credentials
           secret:
             secretName: dapr-trust-bundle
+      affinity:
+        nodeAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 1
+              preference:
+                matchExpressions:
+                - key: kubernetes.io/os
+                  operator: In
+                  values:
+                  - {{ .Values.global.daprControlPlaneOs }}
+            - weight: 2
+              preference:
+                matchExpressions:
+                - key: beta.kubernetes.io/os
+                  operator: In
+                  values:
+                  - {{ .Values.global.daprControlPlaneOs }}
```

Comment:
Why do we need this? do we need to support `beta.kubernets.io/os` ?

Can we just add `if` with `hybrid` value option? I do not think we need to support beta api of k8s cluster. EKS and AKS support `kubernetes.io/os`. I think we can document the supported k8s cluster version instead of supporting beta label.

cc/ @yaron2 @amanbha

Reply:
I don't know necessarily that we do need to support it (I believe it was deprecated in v1.14). But it costs us nothing to have it. I'm happy to remove it, if the consensus is that it should not be there.

Reply:
Oh, I suspect that I misinterpreted our conversation above. Re-reading, I suspect @yaron2 was concerned about the value 'linux' not what version of the kubernetes API we were running against? Or is it both?